pom.xml

```diff
@@ -175,7 +175,7 @@
                (hudi.spark.common.modules.*) -->
     <hudi.spark.common.modules.1>hudi-spark3-common</hudi.spark.common.modules.1>
     <hudi.spark.common.modules.2>hudi-spark3.2plus-common</hudi.spark.common.modules.2>
-    <avro.version>1.8.2</avro.version>
+    <avro.version>1.11.3</avro.version>
```

Comment:
Not sure whether spark-avro work with this, since 1.9, avro introduces many breaking change in their API.

Reply:
@danny0405 any specific workaround to verify if spark-avro is compatible and works fine with avro 1.11.3.

Reply:
We have deprecated Spark 2, 3.0 - 3.2 support, and the minimum Avro version used in Spark is 1.11.2.  So I think this should be fine.

Reply:
then we are fine, Flink also uses avro 1.11+